pkg/apis/configuration/validation/common.go

```diff
@@ -219,3 +219,11 @@ func validateParameter(nPar string, validParams map[string]bool, fieldPath *fiel
 	}
 	return allErrs
 }
+
+func validateEscapedString(body string, examples ...string) error {
```

Comment:
better to move this function right below `var escapedStringsFmtRegexp = regexp.MustCompile("^" + escapedStringsFmt + "$")` so that var is close to the function where it is used. this will be consistent with the rest of the validation code